src/core/api/models/notification_policy.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+import "time"
+
```

Comment:
The actual `NotificationPolicy` struct is in `hook_notification.go` , can we merge this into `hook_notification.go`?

Reply:
Let's move it `src/core/api/notification_policy.go` and make it private.